algoliasearch/src/main/java/com/algolia/search/clients/AnalyticsConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.algolia.search.clients;
+
+import com.algolia.search.models.CallType;
+import com.algolia.search.transport.StatefulHost;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+
+@SuppressWarnings("WeakerAccess")
+public class AnalyticsConfig extends AlgoliaConfig {
+
+  public AnalyticsConfig(String applicationID, String apiKey) {
+    super(applicationID, apiKey);
+
+    List<StatefulHost> hosts =
+        Collections.singletonList(
+            new StatefulHost(
+                "analytics.algolia.com",
```

Comment:
I would use the regional endpoint. analytics.us.algolia.com

As we have more than one